doc/source/whatsnew/v0.24.0.txt

```diff
@@ -405,7 +405,11 @@ Numeric
 
 - Bug in :class:`Series` ``__rmatmul__`` doesn't support matrix vector multiplication (:issue:`21530`)
 - Bug in :func:`factorize` fails with read-only array (:issue:`12813`)
--
```

Comment:
use double-back ticks around TypeError. you don't need the 2nd sentence, just list the issue.